CHANGELOG.md

```diff
@@ -1,102 +1,130 @@
 # pygdbmi release history
 
+## dev
+
+Internal changes
+
+- Update and freeze dependencies for documentation generation
+
 ## 0.10.0.3
```

Comment:
This has not been released yet (the version number was added by the person preparing the PR) so the two sections should be merged.

Sorry, I created a merge conflict when I merged my PR, so this needs to be rebased.